backpack/core/derivatives/mseloss.py

```diff
@@ -1,22 +1,17 @@
-"""Derivatives of the MSE Loss."""
-
+"""NLL extention for Mean Square Error Loss."""
+from abc import ABC
 from math import sqrt
 from typing import List, Tuple
 
-from torch import Tensor, eye, normal, ones
+from torch import Tensor, eye, mean, normal, ones, tensor
+from torch.distributions import Normal
 from torch.nn import MSELoss
 
-from backpack.core.derivatives.basederivatives import BaseLossDerivatives
-
-
-class MSELossDerivatives(BaseLossDerivatives):
-    """Derivatives of the MSE Loss.
+from backpack.core.derivatives.nll_base import NLLLossDerivatives
 
-    We only support 2D tensors.
 
-    For `X : [n, d]` and `Y : [n, d]`, if `reduce=sum`, the MSE computes
-    `∑ᵢ₌₁ⁿ ‖X[i,∶] − Y[i,∶]‖²`. If `reduce=mean`, the result is divided by `nd`.
-    """
```

Comment:
I believe this information is still relevant and should be re-introduced

Reply:
Fixed in comment
@jabader97 [Fixed some commenting, removed mean from mse make distribution, moved…](https://github.com/f-dangel/backpack/pull/252/commits/97cb44f6a85d3bb5b7ec3de309d871fd18f67f8c)